src/main/java/com/greenfox/dramacsoport/petclinicbackend/exceptions/DeletionErrorException.java

```diff
@@ -0,0 +1,11 @@
+package com.greenfox.dramacsoport.petclinicbackend.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.CONFLICT)
+public class DeletionErrorException extends Exception {
```

Comment:
I am not 100% sure, that every deletion error (or exception) is due conflicts. For example if someone would try to delete something which does not exists. It is probably not a conflict. (But this is looking good for now)

Reply:
Yes, you are correct in that case, but not in every case. This is I have concerns adding the status always, I am not against it. For example if there are two admins for example, they are using the UI and clicking on the same Pet clinics deletion. The first one should be processed. But not the other one, because the entity should be not found, but might be still persistent on the site. So my assumption was that the deletions might be different than `CONFLICT`.